src/borg/_hashindex.c

```diff
@@ -445,3 +445,9 @@ hashindex_get_size(HashIndex *index)
 {
     return index->num_entries;
 }
+
+static int
+hashindex_get_disk_size(HashIndex *index)
```

Comment:
the "get disk size" part sounds a bit strange.
how about just "get size" or "size"?